### PR TITLE
CP-1324 Write e2e to cover opening new application windows

### PR DESCRIPTION
**Pull request by rajmundex**
Jira: [CP-1324]

**Description**
Guide: https://appnroll.atlassian.net/wiki/spaces/CP/pages/1681096705/E2E+test+environment+setup+execution+guide+v2

All scenarios are to be skipped until window titles are fixed. 

I added test scenarios to check Help, SAR information, Terms of sercice, License, Privacy Policy - all those are displayed in new window. Basic check is performed against new window - check if specific wrapper component is displayed. 

I also added before section to existing tests as on CI this timeout seems necessary. 

[CP-1324]: https://appnroll.atlassian.net/browse/CP-1324?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/e2e/src/specs/settings/settings-in-app-navigation.e2e.ts [hunk 1]
```diff
@@ -7,8 +7,7 @@ describe("Settings screen check", () => {
     await browser.executeAsync((done) => {
       setTimeout(done, 10000)
     })
-  })
-
+  })ś
```
Comment (igorbezsmertnyi) [inline packages/e2e/src/specs/settings/settings-in-app-navigation.e2e.ts:10]: Let's remove `ś`

Comment (rajmundex) [reply packages/e2e/src/specs/settings/settings-in-app-navigation.e2e.ts]: https://github.com/mudita/mudita-center/pull/766/commits/7e9dfe6f18c7e820657f7b88cbbffa34ca1430ac

**Review by dkarski — COMMENTED** [review]
`wdio.conf.ts` not updated

File: packages/e2e/src/specs/overview/sar-window-check.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import NavigationTabs from "../../page-objects/tabs.page"
+import OverviewPage from "../../page-objects/overview.page"
+
+describe.skip("Check new window display in SAR", () => {
+    before(async () => {
```
Comment (wojcik92michal) [inline packages/e2e/src/specs/overview/sar-window-check.e2e.ts:5]: Formatting

**Review by wojcik92michal — APPROVED** [review]

**Review by igorbezsmertnyi — APPROVED** [review]

**Review by dkarski — APPROVED** [review]

**Pull request merged**